handlers/refunds.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	contentType             = "Content-Type"
+	applicationResponseType = "application/json"
```

Comment:
Would be better named `applicationJsonResponseType`

Reply:
Fixed